tests/it/fix.rs

```diff
@@ -57,6 +57,58 @@ fn fix_output_filters() -> Vec<(&'static str, &'static str)> {
 async fn with_successful_http_no_auth_integration() {
     guard_integration_test!();
 
+    let (_tmp, _, ctx) = temp_config!(load);
+
+    set_snapshot_vars!();
+    let (_, conf) = load_config!(
+        "testdata/config/basic-http-no-auth.yml",
+        "testdata/database/empty.sqlite"
+    )
+    .await;
+
+    let logger = TestLogger::new();
+    broker::cmd::fix::main(&ctx, &conf, &logger, BundleExport::Disable)
+        .await
+        .expect("should run fix");
+
+    insta::with_settings!({ filters => fix_output_filters() },
+       {
+        assert_snapshot!(logger.output());
+       }
+    );
```

Comment:
Prefer to be consistent: either all args are on a newline, or none are (unless rustfmt is doing this, but I think it doesn't try to format macros).
```suggestion
    insta::with_settings!(
        { filters => fix_output_filters() },
        { assert_snapshot!(logger.output()); }
    );
```

Reply:
This was the format of the existing tests. Here is a snippet of the previous code before my code changes:

```
async fn with_successful_http_no_auth_integration() {
    guard_integration_test!();

    set_snapshot_vars!();
    let (_, conf) = load_config!(
        "testdata/config/basic-http-no-auth.yml",
        "testdata/database/empty.sqlite"
    )
    .await;

    let logger = TestLogger::new();
    broker::cmd::fix::main(&conf, &logger, BundleExport::Disable)
        .await
        .expect("should run fix");

    insta::with_settings!({ filters => fix_output_filters() },
       {
        assert_snapshot!(logger.output());
       }
    );
}

#[tokio::test]
async fn with_failing_http_basic_auth_integration() {
    guard_integration_test!();

    set_snapshot_vars!();
    let (_, conf) = load_config!(
        "testdata/config/basic-http-basic-bad-repo-name.yml",
        "testdata/database/empty.sqlite"
    )
    .await;
    let logger = TestLogger::new();
    broker::cmd::fix::main(&conf, &logger, BundleExport::Disable)
        .await
        .expect("should run fix");

    insta::with_settings!({ filters => fix_output_filters() },
       {
        assert_snapshot!(logger.output());
       }
    );
}

#[tokio::test]
async fn with_failing_http_no_auth_integration() {
    guard_integration_test!();

    set_snapshot_vars!();
    let (_, conf) = load_config!(
        "testdata/config/private-repo-http-no-auth.yml",
        "testdata/database/empty.sqlite"
    )
    .await;

    let logger = TestLogger::new();
    broker::cmd::fix::main(&conf, &logger, BundleExport::Disable)
        .await
        .expect("should run fix");

    insta::with_settings!({ filters => fix_output_filters() },
       {
        assert_snapshot!(logger.output());
       }
    );
}
```

I will refactor all the tests to your suggested changes